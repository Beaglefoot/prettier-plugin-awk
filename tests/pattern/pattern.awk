NR==1{}
NR>1,NR<3{}
