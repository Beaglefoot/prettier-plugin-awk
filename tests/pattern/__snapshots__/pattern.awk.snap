NR==1{}
NR>1,NR<3{}
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
NR == 1 {}

NR > 1, NR < 3 {}
